cmd/helm/get_values.go

```diff
@@ -60,6 +62,7 @@ func newGetValuesCmd(client helm.Interface, out io.Writer) *cobra.Command {
 
 	cmd.Flags().Int32Var(&get.version, "revision", 0, "get the named release with revision")
 	cmd.Flags().BoolVarP(&get.allValues, "all", "a", false, "dump all (computed) values")
+	cmd.Flags().StringVar(&get.output, "output", "", "output the specified format (json or yaml)")
```

Comment:
can we change the default output to `"yaml"` rather than the empty string? That way it'll show up as the default in `helm help get values`.

Reply:
Thank you for the review. I have updated the PR.